tools/vscode-azurewebpubsub/src/tree/hubSettings/HubSettingItem.ts

```diff
@@ -0,0 +1,63 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.txt in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import { IActionContext, TreeElementBase, createContextValue, createGenericElement, nonNullValueAndProp } from "@microsoft/vscode-azext-utils";
+import { AzureResourceModel, AzureSubscription, ViewPropertiesModel } from '@microsoft/vscode-azureresources-api';
+import * as vscode from 'vscode';
+import { createWebPubSubAPIClient, localize } from '../../utils';
+import { ServiceItem } from "../service/ServiceItem";
+import { EventHandlersItem } from "./EventHandlers/EventHandlersItem";
+import { EventListenersItem } from "./EventListeners/EventListenersItem";
+import { HubSettingModel, createHubModel } from "./HubSettingModel";
+
+
+export class HubSettingItem implements AzureResourceModel {
+    static readonly contextValue: string = 'webPubSubHubItem';
+    static readonly contextValueRegExp: RegExp = new RegExp(HubSettingItem.contextValue);
+
+    constructor(public readonly service: ServiceItem, public readonly hub: HubSettingModel) { }
+
+    private get contextValue(): string {
+        return createContextValue([
+            HubSettingItem.contextValue,
+            nonNullValueAndProp(this.hub, 'hubName')
+        ]);
+    }
+
+    async getChildren(): Promise<TreeElementBase[]> {
+        const isAllowAnnoy: boolean = this.hub.properties.anonymousConnectPolicy === "allow";
+        const element = createGenericElement({
+            label: localize('allowAnnoyClients', `${isAllowAnnoy ? "Allow" : "Deny"} Anonymous Clients`),
+            contextValue: "hubAllowAnnoymousClients",
+            iconPath: new vscode.ThemeIcon(isAllowAnnoy ? "check" : "error"),
```

Comment:
don't use "error" when deny anonymous? how about some icon meaning trusted?

Reply:
Now icon `workspace-trusted` and `workspace-untrusted` are used for for deny and allow. see https://code.visualstudio.com/api/references/icons-in-labels#icon-listing

Reply:
workspace-untrusted => workspace-unknown?